spec.html

```diff
@@ -70,26 +70,279 @@ <h1>Runtime Semantics: Evaluation</h1>
           1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
           1. Let _specifierString_ be ToString(_specifier_).
           1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
-          1. <ins>Let _attrRef_ be the result of evaluating the second |AssignmentExpression|.</ins>
-          1. <ins>Let _attr_ be ? GetValue(_attrRef_).</ins>
-          1. <ins>If Type(_attr_) is not String,</ins>
-            1. <ins>Call(_promiseCapability.[[Reject]], *undefined*, a new TypeError exception).</ins>
-          1. <ins>Otherwise,</ins>
-            1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, _specifierString_, <ins>_attr_,</ins> _promiseCapability_).
+          1. <ins>Let _argRef_ be the result of evaluating the second |AssignmentExpression|.</ins>
+          1. <ins>Let _arg_ be ? GetValue(_argRef_).</ins>
+          1. <ins>If _arg_ is *undefined*, let _attributes_ be *undefined*.</ins>
```

Comment:
Note that this PR also fixes #52 .

Reply:
Good point, added to the commit message.